eZ/Publish/API/Repository/Tests/ContentServiceTest.php

```diff
@@ -2696,23 +2696,41 @@ public function testLoadContentWithSecondParameter()
 
         $this->assertLocaleFieldsEquals($draftLocalized->getFields(), self::ENG_GB);
 
-        return $draft;
+        return $draftLocalized;
     }
 
     /**
      * Test for the loadContent() method using undefined translation.
      *
-     * @depends eZ\Publish\API\Repository\Tests\ContentServiceTest::testLoadContentWithSecondParameter
+     * @depends eZ\Publish\API\Repository\Tests\ContentServiceTest::testLoadContentWithPrioritizedLanguages
      *
      * @param \eZ\Publish\API\Repository\Values\Content\Content $contentDraft
      */
-    public function testLoadContentWithSecondParameterThrowsNotFoundException(Content $contentDraft)
+    public function testLoadContentWithPrioritizedLanguagesThrowsNotFoundException(Content $contentDraft)
     {
         $this->expectException(NotFoundException::class);
 
         $this->contentService->loadContent($contentDraft->id, [self::GER_DE], null, false);
     }
 
+    /**
+     * Test for the loadContent() method.
+     *
+     * @see \eZ\Publish\API\Repository\ContentService::loadContent($contentId, $languages)
```

Comment:
no need for arguments here
```suggestion
     * @see \eZ\Publish\API\Repository\ContentService::loadContent
```

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/2833/commits/b6ec7f1abcfe45179630d318fb5e9165c4031489